src/main/java/org/craftercms/studio/api/v2/dal/QueryParameterNames.java

```diff
@@ -265,6 +246,8 @@ public final class  QueryParameterNames {
 
     public static final String SYSTEM_TYPE_FOLDER = "systemTypeFolder";
 
+    public static final String PREFER_CONTENT = "preferContent";
```

Comment:
Are we keeping this feature?

Reply:
yes, turns out it is used not only by the UI but the backend too, so I think it's better to keep it for now